tapdb/universe_stats.go

```diff
@@ -141,6 +141,7 @@ func (u *UniverseStats) LogSyncEvent(ctx context.Context,
 			EventTimestamp: u.clock.Now().UTC().Unix(),
 			AssetID:        uniID.AssetID[:],
 			GroupKeyXOnly:  groupKeyXOnly,
+			ProofType:      uniID.ProofType.String(),
```

Comment:
Would've expected these test run on top of that branch to trigger unit test failures within it?

Reply:
Yes me too, but the stats must be aggregated correctly regardless, or we were missing coverage.